### ISSUE TITLE
typo auf https://codefor.de/projekte/fis-broker/

### ISSUE THREAD

**Issue by Wikinaut** [codeforberlin/luftbilder.berlin.codefor.de#17]
s/habe Teile/haben Teile/

**Comment by jochenklar** [comment]
Danke! Wird gefixed.

**Comment by Wikinaut** [comment]
Anders:
Wo finde ich  denn diese frühere Übersicht (Karte pro Karte mit jeweils einem Beispiel) – gibt es die noch?

Und bei https://luftbilder.berlin.codefor.de/ fehlt mir jeweils über oder unter den Auswahlflächen direkt der Link auf die entsprechende Serverurl am besten in der Form für Bikerouter und ähnliche Anwendungen

![grafik](https://github.com/codeforberlin/luftbilder.berlin.codefor.de/assets/1151915/d8bcaa21-fed2-4ba7-9e60-381e78b42f56)

**Comment by Wikinaut** [comment]
Ach, das hab ich gesehen.

change request hier:

s/Um die Karte/Um **_diese_** Karte/

**Comment by Wikinaut** [comment]
> Das gibt es auf https://maps.berlin.codefor.de/ unter dem Fragezeigen oben links. Vll sollte ich das alles wieder zusammenführen...

JA, bitte unbedingt, ich bin gestern vollkommen immer hin und her gelaufen, too many pages..... looking similar, but were different.

Wir können auch direkt kommunizieren, tordans müsste meine Kontaktdaten haben.
